### ISSUE TITLE
XEMM - Getting 'NoneType' object has no attribute 'creation_timestamp' on "history --verbose"

### ISSUE THREAD

**Issue by rapcmia** [hummingbot/hummingbot#5284]
**Describe the bug**
For tracking purpose
While checking Cross exchange market making strategy, there are instances of the bot getting `'NoneType' object has no attribute 'creation_timestamp'` when running` history --verbose` command after few trades. Intially it was spotted when using Gateio and Ascendex however when tested with other connectors e.g Gateio and Crypto_com, the issue does not happen

```
2022-05-04 18:45:14,770 - 17808 - hummingbot.client.hummingbot_application - ERROR - 'NoneType' object has no attribute 'creation_timestamp'
Traceback (most recent call last):
  File "/home/rapcmia/github/dev/hummingbot/client/hummingbot_application.py", line 205, in _handle_command
    f(**kwargs)
  File "/home/rapcmia/github/dev/hummingbot/client/command/history_command.py", line 62, in history
    self.list_trades(start_time)
  File "/home/rapcmia/github/dev/hummingbot/client/command/history_command.py", line 230, in list_trades
    df: pd.DataFrame = TradeFill.to_pandas(queried_trades)
  File "/home/rapcmia/github/dev/hummingbot/model/trade_fill.py", line 124, in to_pandas
    age = pd.Timestamp(int(trade.timestamp / 1e3 - trade.order.creation_timestamp / 1e3),
AttributeError: 'NoneType' object has no attribute 'creation_timestamp'
```

![image](https://user-images.githubusercontent.com/73840223/166667498-834f78f0-d92b-4f07-9a9c-2327b0be92af.png)

**Steps To Reproduce**
1. Create simple XEMM strategy
2. Setup min_profitability to 0.001 or lower enough to trigger trades
3. Start and monitor bot performance 
4. After few trades, run `history --verbose`

**Release version**
development branch (dev-1.4)

**Attachments**
[xemm-gateio-ascendex.zip](https://github.com/hummingbot/hummingbot/files/8619621/xemm-gateio-ascendex.zip)

**Comment by nikspz** [comment]
Fixed on development 1.8.0

Tested using huobi-ftx and also few trades on gate.io-ascendex
![image](https://user-images.githubusercontent.com/83953535/192869126-ac4c088d-8384-407f-8fea-1d6b89eb23f0.png)

**Comment by nikspz** [comment]
Reproduced again on dev-1.10.0, the issue is floating (not happened all the time)
